code/game/objects/items/spraypaint.dm

```diff
@@ -0,0 +1,48 @@
+/obj/item/spraypaint
+	name = "spraycan"
+	desc = "Aerosol paint sealed in a pressurized container."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "spraypaint"
+	matter = list(MATERIAL_PLASTIC = 4, MATERIAL_STEEL = 5)
+	price_tag = 200
+	var/uses = 30
+	w_class = ITEM_SIZE_TINY
```

Comment:
```suggestion
	w_class = ITEM_SIZE_SMALL
```
Tiny is something that logically should be able fit behind ears. Definitely not possible with a spraycan